tests/e2e/executors/Executor.java

```diff
@@ -25,9 +25,7 @@ public static void main(String[] args) {
                 String out = "";
                 for (int i = 0, l = arr.length; i < l; i++) {
                     out += arr[i];
-                    if (i < l - 1) {
```

Comment:
Why this has been removed? Now it'll output the unnecessary trailing whitespace.